helpers/presentation_objects.py

```diff
@@ -98,6 +98,15 @@ def load(self, dst_folder, **kwargs):
 
         fig = plt.figure(figsize=(6, 4), dpi=150)
         ax = fig.add_subplot(1, 1, 1)
+
+        # show a (constant) reference value if it is defined
+        reference = kwargs.get("reference", None)
+        if reference:
+            ref_value = reference['value']
+            ref_label = reference['label']
+            ax.plot([coord_points[0],coord_points[-1]],[ref_value,ref_value],
```

Comment:
I suggest `ax.axhline(y=ref_value, color="r", linestyle=":")` instead. See [Matplotlib documentation](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.axhline.html) for more info

Reply:
Good idea, fixed in aea16f58